Intersect.Server/Networking/PacketSender.cs

```diff
@@ -602,6 +602,34 @@ public static void SendChatMsg(Player player, string message, ChatMessageType ty
             SendChatMsg(player, message, type, CustomColors.Chat.PlayerMsg, target);
         }
 
+        /// <summary>
+        /// Logs a chat message if chat message logging is enabled
+        /// </summary>
+        /// <param name="player">The player to which to send a message.</param>
+        /// <param name="message">The message to send.</param>
+        /// <param name="type">The type of message we are sending.</param>
+        /// <param name="target">The sender of this message, should we decide to respond from the client.</param>
+        public static void LogChatMsg(Player player, string msg, ChatMessageType type)
```

Comment:
I would greatly prefer this being moved into `ChatHistory` as a static method at the bottom of that file.